trpc/router/goal.ts

```diff
@@ -393,15 +400,15 @@ export const goal = router({
                 });
             }
 
-            if (actualGoal.priority !== input.priority) {
+            if (previousPriority && actualPriority && previousPriority?.id !== actualPriority?.id) {
                 history.push({
                     subject: 'priority',
                     action: 'change',
-                    previousValue: actualGoal.priority,
-                    nextValue: input.priority,
+                    previousValue: previousPriority.title,
```

Comment:
Why not priorityId?

Reply:
in GoalActivity we pass strings to show what has changed to what